research-hub-web/cypress/integration/equipment.e2e.js

```diff
@@ -0,0 +1,65 @@
+describe('ResearchHubs Equipment  Pages', () => {
+
+    beforeEach(() => {
+        cy.visit('/equipment/epson-surecolor-p10070');
+    });
+
+    it('can visit an equipment and display its title', () => {
+        cy.contains('Epson SureColor P10070');
+    });
+
+    it('can visit an equipment and display its subtitle', () => {
+        cy.contains('The SureColor Production 10070 is a 111.8cm (44 inch) Large Format Printer ideal for fine art printing, photos, high resolution images.');
+    });
+
+    it('equipment displays body text', () => {
+        cy.contains('Please take care with submitting your work. All files will be printed to these specifications alone - missed information may result in your file not being printed.');
+    });
+
+    it('equipment displays specifications table', () => {
+        cy.get('#specifications-table').contains('Details').should('exist');
+        cy.get('#specifications-table').contains('Description').should('exist');
+    });
+
+    it('equipment displays specifications table 2', () => {
+        cy.get('#specifications-table2').contains('Details').should('exist');
+        cy.get('#specifications-table2').contains('Description').should('exist');
+    });
+
+    it('displays a list of related items', () => {
+        cy.get('#you-might-be-interested-in').should('exist');
+    });
+
+    it('clicking a related item takes you to its page', () => {
+        cy.get('#you-might-be-interested-in').contains('Digital Print Centre').click();
+        cy.contains('The Creative Arts and Industries Print Centre is a unique large format print service available to students and staff across the whole University.');
+    });
+
+    it('displays a list of contacts', () => {
+        cy.get('#contacts').contains('Vicki Ormond').should('exist');
+    });
+    
+    // No document in equipment
+    //
+    // it('displays a list of documents', () => {
+    //     cy.get('#documents').contains('IT Acceptable Use Policy').should('exist');
+    // });
+    
+    // it('clicking on a documents takes you to the documents', () => {
+    //     cy.get('#documents').contains('Health Research Council – Research Impact Guidance').click();
+    //     cy.contains('The Impact of Research');
+    // });
+
+    // No Organisations in equipment
+    // 
+    // it('displays a list of organisations', () => {
+    //     cy.get('#organisations').contains('Centre for eResearch').should('exist');
+    // });
+
+    // Cypress doesn't dupport multi-tab testing
+    // 
```

Comment:
some options here: https://filiphric.com/opening-a-new-tab-in-cypress 
really we just want to check link is valid? So you could use something like this as suggested in that article: 
    cy
      .request(link)
      .its('status')
      .should('eq', 200);
  });

Reply:
Having said that.. Cam was planning to have a link checker service that will be checking all our external links are valid, so maybe doing that in our e2e tests is unnecessary

Reply:
Maybe the best thing is just to check that there is actually a link present